### ISSUE TITLE
Idle notifications are not shown when one stops mining Dense essence blocks for Blood/Soul Runecrafting

### ISSUE THREAD

**Issue by RaspberrySchnapps** [runelite/runelite#5214]
**What happened?**
When one stops mining Dense essence blocks used for Blood and Soul runecrafting, idle notifications are not displayed anymore since today's update(30th of August)

**How to reproduce**
1. Go to Arceuus essence mine
2. Make sure the 'Idle Notifier' plugin is enabled and has the 'Idle Animation Notifications' checkbox checked
3. Start mining the Dense runestone until it depletes or the inventory fills up

**Expected result:** the Idle Notifier alerts that the mining action has stopped
**Actual result:** the Idle Notifier does not alert that the mining action has stopped

**OS:** Windows 7
**RuneLite version:** 1.4.15
**Oldschool revision:** Rev 174

**Comment by deathbeam** [comment]
Duplicate of #836

**Comment by RaspberrySchnapps** [comment]
I urge you to reconsider. I've found that thread prior to posting this one. Although I might be in the wrong here, the parent case is a few months old, with the latest comment of 29 days implying that they "could not reproduce". In addition, the idle alert was working as expected for a few days at the very least(as I only started training RC there recently) before today's update of Runelite.
